### ISSUE TITLE
[TW-1836] `task info` shows stripped description for annotated tasks

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#1852]
_Klas Š. on 2016-08-01T09:11:47Z says:_ 

 Two other condition of reproducing the bug:
 * description should be in Cyrillic (probably will work with any non-ASCII alphabet/writing system)
 * task should be urgent enough to be highlighted.

Steps  to reproduce:
{code}
 % task add "Съесть ещё этих мягких французских булок"
Created task 1.

 % task 1 mod pri:H
Modifying task 1 'Съесть ещё этих мягких французских булок'.
Modified 1 task.

 % task 1 info

Name          Value
ID            1
Description   Съесть ещё этих мягких французских булок
Status        Pending

 % task 1 annotate "Да выпить чаю"
Annotating task 1 'Съесть ещё этих мягких французских булок'.
Annotated 1 task.

 % task 1 info

Name          Value
ID            1
Description   Съесть ещё этих мягки�
                2016-08-01 11:58:45 Да выпить чаю
Status        Pending

 % task 1 mod pri:
Modifying task 1 'Съесть ещё этих мягких французских булок'.
Modified 1 task.

 % task 1 info

Name          Value
ID            1
Description   Съесть ещё этих мягких французских булок
                2016-08-01 11:58:45 Да выпить чаю
Status        Pending
{code}

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2016-08-01T09:11:47Z
Modified: 2017-10-12T02:57:45Z
```

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2016-08-01T22:42:33Z says:_ 

 Similar problem, different use case.

**Comment by grayed** [comment]
Just got a similar same problem on 2.5.1, but when using tagged task instead of prioritized one. It looks like somewhere byte-specific routine is called instead of character-specific, since Cyrillic UTF-8 characters are 2 bytes long.

**Comment by Pastafarianist** [comment]
Also reproduced this on 2.5.1.

**Comment by laeder** [comment]
I have the same problem using Swedish, discovered it today. Every line in Description is missing 1 or more characters except the last one. The first line should end with the word 'börjar'. I'm also using 2.5.1.

```
Name          Value                                                      
ID            6
Description   Håll koll på när NCIS Los Angeles bö                       
                2019-06-13 19:46:12 Även vanliga NCI                     
                2019-06-14 07:56:47 NCIS börjar 24 septembe              
                2019-06-14 09:52:21 NCIS: Los Angeles börjar 29 september
Status        Pending
Entered       2019-06-10 20:20:32 (3d)                                   
Scheduled     2019-09-01 00:00:00
Last modified 2019-06-14 09:52:21 (46min)                                
Virtual tags  ANNOTATED PENDING SCHEDULED UNBLOCKED
UUID          f269de8b-8b93-4540-b995-81e5216bf1ca                       
Urgency       1.016

    annotations      1 *    1 =      1
    age          0.008 *    2 =  0.016
                                ------
                                 1.016

Date                Modification                                                
2019-06-13 19:46:12 Annotation of 'Även vanliga NCIS' added
2019-06-14 07:56:47 Annotation of 'NCIS börjar 24 september' added              
2019-06-14 09:52:21 Annotation of 'NCIS: Los Angeles börjar 29 september' added.

```

**Comment by Dahko** [comment]
+1 here. To add some details: MacOS Catalina, taskwarrior 2.5.1 via Homebrew, reproduces on both system Terminal and iTerm.
Is there maybe an insight where to look in the code to try fixing this myself?

**Comment by tbabej** [comment]
Confirming this can be reproduced on 2.5.1, but is fixed on 2.5.3 or later.

```
[warrior@tw-v2.5.3 ~]$ task add Håll koll på när NCIS Los Angeles börjar
Created task 1.
[warrior@tw-v2.5.3 ~]$ task 1 ann Även vanliga NCIS
Annotating task 1 'Håll koll på när NCIS Los Angeles börjar'.
Annotated 1 task.
[warrior@tw-v2.5.3 ~]$ task 1 info

Name          Value                                    
ID            1                                        
Description   Håll koll på när NCIS Los Angeles börjar
                2021-06-27 03:49:17 Även vanliga NCIS
```
